doc/api/deprecations.md

```diff
@@ -994,6 +994,14 @@ because it also made sense to interpret the value as the number of bytes
 read by the engine, but is inconsistent with other streams in Node.js that
 expose values under these names.
 
+<a id="DEPXXXX"></a>
+### DEPXXXX: vm.Script cached data
+
+Type: Documentation-only
+
+The options `produceCachedData` and `cachedData` have been deprecated. Please use
```

Comment:
I would also support splitting the deprecation into a separate PR, so that we can ship this earlier without semver changes.

Reply:
I changed it to docs only, not sure why I made it runtime in the first place

Reply:
Even as a docs-only deprecation, it should be separated out into a separate commit.

Reply:
Also ... just a nit... s/DEPXXXX/DEP00XX ... the release tooling looks specifically for the `DEP00XX` pattern to ensure that the codes are properly assigned before release.